examples/chatgpt/rag-quickstart/azure/Azure_AI_Search_with_Azure_Functions_and_GPT_Actions_in_ChatGPT.ipynb

```diff
@@ -0,0 +1,1330 @@
+                "                  description: category to filter.\n",
+                "              required:\n",
+                "                - search_service_endpoint\n",
+                "                - index_name\n",
+                "                - query\n",
+                "                - k_nearest_neighbors\n",
+                "                - search_column\n",
+                "                - use_hybrid_query\n",
+                "      responses:\n",
+                "        '200':\n",
+                "          description: A successful response with the search results.\n",
+                "          content:\n",
+                "            application/json:\n",
+                "              schema:\n",
+                "                type: object\n",
+                "                properties:\n",
+                "                  results:\n",
+                "                    type: array\n",
+                "                    items:\n",
+                "                      type: object\n",
+                "                      properties:\n",
+                "                        id:\n",
+                "                          type: string\n",
+                "                          description: The identifier of the result item.\n",
+                "                        score:\n",
+                "                          type: number\n",
+                "                          description: The similarity score of the result item.\n",
+                "                        content:\n",
+                "                          type: object\n",
+                "                          description: The content of the result item.\n",
+                "        '400':\n",
+                "          description: Bad request due to missing or invalid parameters.\n",
+                "        '500':\n",
+                "          description: Internal server error.\n",
+                "\"\"\"\n",
+                "    pyperclip.copy(spec)\n",
+                "    print(\"OpenAPI spec copied to clipboard\")\n",
+                "    print(spec)\n",
+                "generate_openapi_spec(app_name)"
+            ]
+        },
+        {
+            "cell_type": "markdown",
+            "metadata": {},
+            "source": [
+                "### 2. Create a GPT with an action that is the above, and the below instructions\n",
+                "\n",
+                "Feel free to modify instructions as you see fit. There are a few techniques you can take below, including re-ranking, guardrails, metadata filtering, etc. to name a few."
+            ]
+        },
+        {
+            "cell_type": "code",
+            "execution_count": null,
+            "metadata": {},
+            "outputs": [],
+            "source": [
+                "\n",
+                "def generate_and_copy_instructions(search_service_endpoint, index_name, categories):\n",
```

Comment:
same as above, not sure what the value of this function is since we aren't going to be re-using it